README.md

```diff
@@ -321,7 +321,7 @@ It does not work with some features. Also, if you change the default behaviour,
 
 ### Custom build
 
-For some cases could be useful adding a blacklist of features or generation a polyfill for target engines. You could use [`core-js-builder`](/packages/core-js-builder) package for that.
+For some cases could be useful adding a _exclude_ of features or generation a polyfill for target engines. You could use [`core-js-builder`](/packages/core-js-builder) package for that.
```

Comment:
Please, rephrase into something more readable. The rest entries - too.

Reply:
@zloirock what do you think about the following:


> For some cases could be useful generating a especific polyfill for target engines.
>
> You can use [`core-js-builder`](/packages/core-js-builder) package for include or exclude parts of `core-js`. And optionally generate a file from that.